modules/logging/filewriter.go

```diff
@@ -96,6 +98,15 @@ type FileWriter struct {
 	// it will be rotated.
 	RollSizeMB int `json:"roll_size_mb,omitempty"`
 
+	// Roll log file after some time
+	RollInterval time.Duration `json:"roll_interval,omitempty"`
+
+	// Roll log file at fix minutes
+	RollAtMinutes []int `json:"roll_at_minutes,omitempty"`
+
+	// Roll log file at fix time
+	RollAt []string `json:"roll_at,omitempty"`
```

Comment:
These godoc comments should cover some of the caveats mentioned in the timberjack README https://github.com/DeRuina/timberjack#%EF%B8%8F-rotation-notes--warnings (and eventually also in the Caddy website docs)